js/blogs_info/blogs_info.js

```diff
@@ -1,6 +1,57 @@
 var blogs = [];
 
 /* 
-To add a new blog, add a new line in this format: "blogs.push(new blog('blog name', 'blog imageLocation', 'blog_link');"
+To add a new blog, add a new item to blogsInfo and push that item to blogs.
 */
-blogs.push(new blog("Gav's Dev Blog", "./images/blog_images/gavs_dev_blog.jpg", "https://gavsdevblog.wordpress.com/"));
+
+var blogsInfo = {
+  'loriansBlog': {
+    'name': "Lorian Bates",
+    'imageLocation': '',
+    'link': 'https://locodesblog.wordpress.com/2016/08/04/polymorphic-me/'
+  },
+  'mckinnelysBlog': {
+    'name': "McKinnely Bentley",
+    'imageLocation': '',
+    'link': 'http://mbcodeguy.blogspot.com/'
+  },
+  'matthewsBlog': {
+    'name': "Matthew David",
+    'imageLocation': '',
+    'link': 'https://mattattaqdevblog.wordpress.com/'
+  },
+  'gwensBlog': {
+    'name': "Gwendolyn Faraday",
+    'imageLocation': '',
+    'link': 'http://gwenfaraday.com/LearningCode/'
+  },
+  'nicholasBlog': {
+    'name': "Nicholas Gillespie",
+    'imageLocation': '',
+    'link': 'http://programpractical.com/'
+  },
+  'gavynsBlog': {
+    'name': "Gavyn Leavitt",
+    'imageLocation': './images/blog_images/gavs_dev_blog.jpg',
+    'link': 'https://gavsdevblog.wordpress.com/'
+  },
+  'jtsBlog': {
+    'name': "J.T. Maupin",
+    'imageLocation': '',
+    'link': 'https://jmsmaupin.wordpress.com/'
+  },
+  'kimberlysBlog': {
+    'name': "Kimberly Sharpe",
+    'imageLocation': '',
+    'link': 'http://kimmihack.blogspot.com/'
+  },
+  'larrysBlog': {
+    'name': "Larry Tooley",
+    'imageLocation': '',
+    'link': 'http://damagedtechnologies.com'
+  }
+};
```

Comment:
Though this works, we have no need to call these objects by a name (`larrysBlog`), as they are just looped through and there is no additional information gained by having that since they are already contain a name field (`'name': 'Larry Tooley',`) and we know these are all blogs because they are in a variable called `blogs`.

So instead, it could just be:

```
var blogList = [
    {
        'name': 'Larry Tooley',
        'image': '',
        'url': 'http://damagedtechnologies.com'
    },
    {
        'name': 'Kimberly Sharpe',
        'image': '',
        'url': 'http://kimmihack.blogspot.com'
    }
];

for (var i = 0; i < blogList.length; i++) {
    blogs.push(new blog(blogList[i]));
}
```

This is only slightly cleaner, normally I wouldn't nitpick to this level and would just merge this in as it works perfectly fine. But since the point of this project is to be a learning experience, I thought I'd throw this out there so when you deal with structuring your data in the future with something more complex, you'll be able to think about it from more angles.